pkg/webhooks/subscription/subscription.go

```diff
@@ -122,7 +122,7 @@ func (s *SubscriptionWebhook) authorized(request admissionctl.Request) admission
 	}
 
 	// can comment this out or remove after manual tests
-	log.Info("User is attempting to modify subscription", "username", request.AdmissionRequest.UserInfo.Username, "operation", request.Operation, "subscription name", subReq.Spec.Name, "channel", subReq.Spec.Channel)
+	// log.Info("User is attempting to modify subscription", "username", request.AdmissionRequest.UserInfo.Username, "operation", request.Operation, "subscription name", subReq.Spec.Name, "channel", subReq.Spec.Channel)
```

Comment:
```suggestion
```

Any reason not to 🔪 this entirely?

Reply:
I figured I'd leave it in to save some time in case future me (or future someone else) needs to debug this webhook

Reply:
Okay. I half expect someone else coming through the code to go "oh this is odd" and 🔪 it, so fair warning I suppose? 🤷‍♀️ 

/lgtm